ql/cashflows/fixedratecoupon.cpp

```diff
@@ -176,7 +176,16 @@ namespace QuantLib {
                                                      exCouponEndOfMonth_);
         }
 
-        if (schedule_.isRegular(1)) {
+        if(!schedule_.hasIsRegular() || !schedule_.hasTenor()) {
```

Comment:
It might be a good idea to have some comment here explaining what kind of fallback we're implementing...